.github/workflows/autoformat.yml

```
@@ -151,6 +151,7 @@ jobs:
       - name: Create patch
         id: patch
         run: |
+          git add --intent-to-add .
```

```suggestion
          git add -N .
``` 
let's use short form